src/test/groovy/life/qbic/datamodel/datasets/datastructure/OxfordNanoporeExperimentSpec.groovy

```diff
@@ -79,6 +89,20 @@ class OxfordNanoporeExperimentSpec extends Specification {
         assert measurements[0].libraryPreparationKit == "SQK-LSK109-XL"
     }
 
+    def "Create extended sample Oxford Nanopore experiment successfully for newer structure"() {
+        given:
+        final def example = extendedWorkingSimpleDataStructureWithReportsFolder
+
+        when:
+        final def experiment = OxfordNanoporeExperiment.create(example)
+        final def measurements = experiment.getMeasurements()
+
+        then:
+        assert experiment.sampleCode == "QABCD001AB"
+        assert measurements.size() == 1
+        assert measurements[0].libraryPreparationKit == "SQK-LSK109-XL"
```

Comment:
Nit: You could also directly test for specific attributes of the extendedWorkingSimpleDataStructureWithReportsFolder here to make the test more specific.